js/components/ReportSummary.vue

```diff
@@ -33,25 +35,104 @@ export default {
   },
 
   methods: {
+    /**
+     * Emit deleteSummary event.
+     */
     deleteSummary() {
       if (confirm('Permanently delete this summary?')) {
         this.$emit('deleteSummary', this.index);
       }
     },
 
+    /**
+     * Checks if the user has permission to delete a report summary.
+     * @return true if the user has permission to delete a report summary.
+     */
     canDelete() {
       // TODO: implement REDDEV-595
       return true;
+    },
+
+    /**
+     * Checks a value to determine if it is missing. Missing values are those
+     * that are zero in length, undefined, only whitespace, or null.
+     * @return true if the value is missing
+     */
+    missingValue(value) {
+      if (value == null || typeof value === 'undefined') {
```

Comment:
Any reason this is `value == null` instead of `value === null`? You can also compare to undefined without using typeof.

Reply:
It also looks like you're missing config for ESLint, because it would definitely have flagged this.